app/controllers/spree/retail/shopify/hooks/refunds_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Spree
+  module Retail
+    module Shopify
+      module Hooks
+        class RefundsController < HooksController
+          def create
```

Comment:
In the Shopify UI, you can't re-refund something that has already been re-fund. 

I am unsure of the case where we would receive twice the same event and thus having to manage the case where it has to be idempotent. But I could take a look and make it work if you think it's a good idea.